docs/release_notes/v0.4.0.md

```diff
@@ -0,0 +1,267 @@
+
+# Dapr 0.4.0
+ 
+We're happy to announce the release of Dapr 0.4.0! 
+
+This release includes new components, security improvements, API level enhancements, new CLI features, stability and bug fixes, a richer Java SDK, docs and samples updates. 
+
+We would like to extend our thanks to all new and existing contributors who helped make this release happen.
+
+If you're new to Dapr, visit the [getting started](https://github.com/dapr/docs/tree/master/getting-started) page and familiarize yourself with Dapr.
+
+*__Note: This release of Dapr contains breaking changes. Read through for more details, and see [this](#upgrading-to-dapr-0.4.0) section on upgrading Dapr__*
+
```

Comment:
I would mention breaking change or its link in the beginning because users are unlikely to read all release notes.

Reply:
Done